docs/references/ayx-python-sdk-v2-example-tools.md

```diff
@@ -0,0 +1,108 @@
+# AYX Python SDK v2 Example Tools
+
+### Example Tools
+
+These are example tools created using version 2 the [AYX Python
+SDK](https://help.alteryx.com/developer-help/ayx-python-sdk). When the tools are installed, they display in the SDK
```

Comment:
Point this to the new reference on ayx-developer-sdk repo.

This should point to the corresponding markdown version